app/dashboard/src/services/RemoteBackend.ts

```diff
@@ -1047,7 +1052,10 @@ export default class RemoteBackend extends Backend {
   override async listUserGroups(): Promise<backend.UserGroupInfo[]> {
     const path = remoteBackendPaths.LIST_USER_GROUPS_PATH
     const response = await this.get<backend.UserGroupInfo[]>(path)
-    if (!responseIsSuccessful(response)) {
+
```

Comment:
blank line. i think at least we should be consistent whether or not to have  a blank line